### PR TITLE
Update README.md for master binaries plug

### PR DESCRIPTION
**Pull request by michgauz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,32 @@ Current integrations include:
 * Transifex
 * Slack
 
+# Plug on Snapshots
+
+Install kinta from master:
```
Comment (martinbonnin) [inline README.md:54]:
```suggestion
# Snapshots

You can install the latest build from the master branch with the following command:
```

File: README.md [hunk 2]
```diff
@@ -49,6 +49,32 @@ Current integrations include:
...
+
+```
+curl -s 'https://dailymotion.github.io/kinta/install_from_master.sh' |sh
```
Comment (martinbonnin) [inline README.md:57]: Maybe rename to `install_snapshot.sh` ?

File: README.md [hunk 3]
```diff
@@ -49,6 +49,32 @@ Current integrations include:
...
+
+```
+
+If you have a `kintaSrc` project :
```
Comment (martinbonnin) [inline README.md:61]: 👍

**Review by martinbonnin — APPROVED** [review]
A few comments and suggestions. Nothing blocking, up to you to take them or throw them away!

File: README.md (hunk 2 above)
Comment (michgauz) [reply README.md:57]: ok

**Pull request merged**